package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, jaclark5
+??/??/?? IAlibay, jaclark5, MohitKumar020291
```

Comment:
I fixed the CHANGELOG for you. You needed to merge develop into your PR.

For doc test fixes and similar changes that are not user visible we typically only record you as a contributor with your GitHub handle @MohitKumar020291 and don't list the specific change.

You don't have to make any more changes in CHANGELOG.